lib/poller.js

```diff
@@ -69,9 +115,40 @@ function updateCoinAtmRadar () {
     .then(rates => coinAtmRadar.update(rates, settings()))
 }
 
-function start (__settings) {
-  reload(__settings)
+function initializeEachSchema (schemas = ['public']) {
+  // for each schema set "thread variables" and do polling
+  return _.forEach(schema => {
+    const store = defaultStore()
+    store.set('schema', schema)
+    return asyncLocalStorage.run(store, () => {
+      return settingsLoader.loadLatest().then(settings => {
+        // prevent inadvertedly clearing the array without clearing timeouts
+        if (schemaCallbacks.has(schema)) throw new Error(`The shema "${schema}" cannot be initialized twice on poller`)
```

Comment:
You have `schema` misspelled

```suggestion
        if (schemaCallbacks.has(schema)) throw new Error(`The schema "${schema}" cannot be initialized twice on poller`)
```